webnn/resources/utils.js

```diff
@@ -1,5 +1,21 @@
+/**
+ * Copyright 2021 WebNN Tests contributors
```

Comment:
@dontcallmedom 

From https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/ops/broadcast_util.ts#L3
```
Copyright 2017 Google LLC. All Rights Reserved.
```

, I'm not sure that it's allowable to use the code of tensorflow/tfjs project. Any comment, thanks.

Reply:
Since the files we're copying ares under Apache 2 license, as long as we keep the licensing info intact, there should be no issue with copying the code. But the copyright needs to remain to Google LLC (here and in all the other copied files)